README.md

```diff
@@ -13,34 +25,28 @@ A collection may contain metadata that identifies these versions.
 PEP440 is the schema used to describe the versions of Ansible.
 <!--end requires_ansible-->
 
-## Python version compatibility
+### Python version compatibility
 
 This collection requires Python 3.9 or greater.
 
-## AWS SDK version compatibility
+### AWS SDK version compatibility
 
 Version 0.3.0 of this collection supports `boto3 >= 1.25.0` and `botocore >= 1.28.0`
 
-## Included content
-<!--start collection content-->
+## Installation
 
-See the complete list of collection content in the [Plugin Index](https://ansible-collections.github.io/amazon.cloud/branch/main/collections/amazon/cloud/index.html#plugin-index).
-
-<!--end collection content-->
-
-## Installing this collection
-
-You can install the AWS Cloud Control Collection with the Ansible Galaxy CLI:
+A specific version of the AWS Cloud Control Collection can be installed with Ansible Galaxy command-line tool:
 ```bash
-    ansible-galaxy collection install amazon.cloud
+    ansible-galaxy collection install amazon.cloud==1.0.0
 ```
 
-You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+It can also included in a `requirements.yml` file and installed with `ansible-galaxy collection install -r requirements.yml`, using the format:
```

Comment:
```suggestion
It can also be included in a `requirements.yml` file and installed with `ansible-galaxy collection install -r requirements.yml`, using the following format:
```